### PR TITLE
Create endpoint for running compliance checker

### PR DESCRIPTION
**Pull request by Bobfrat**
@lukecampbell please review. Very simple at the moment, jsut returns the json response (or redirects to report page if report_type=html but I'm not sure how useful that is).

I'll work on the documentation next. Wiki page on this repo ok?

### CODE REVIEW

**Comment by Bobfrat** [comment]
example:
```
http://localhost:3000/api/run?report_format=json&test=cf&url=http://chlthredds.erdc.dren.mil/thredds/dodsC/frf/oceanography/ctd/eop-ctd/2014/FRF-ocean_ctd_eop-ctd_201405.nc
```

File: cchecker_web/api.py [hunk 1]
```diff
@@ -32,6 +37,76 @@ def get_tests():
...
+    required_fields = [test, url]
+    if all(required_fields):
+        # Kick off the job
+        job_id = get_job_id(url)
+        app.queue.enqueue_call(func=compliance_check, args=(job_id, url, test))
+        job_result = None
+        timeout = 20  # secs
+
+        # Now we check to see if processing is done
+        for tries in xrange(timeout):
```
Comment (lukecampbell) [inline cchecker_web/api.py]: please use `range` instead of `xrange` as `xrange` isn't in python3

File: cchecker_web/api.py [hunk 2]
```diff
@@ -32,6 +37,76 @@ def get_tests():
...
+    report_format = request.args.get('report_format', 'json')
+    test = request.args.get('test', None)
+    url = request.args.get('url', None)
+
+    # Check report formats
+    accepted_formats = ['json', 'html']
+    if report_format not in accepted_formats:
+        err_msg = ("Report format '{0}' not available. "
+                   "Please choose from {1}".format(report_format, accepted_formats))
+        return jsonify({'Error': err_msg}), 400
```
Comment (lukecampbell) [inline cchecker_web/api.py]: is this the same style of error reporting I do in the other methods? I typically keep error lowercase.

Comment (Bobfrat) [inline cchecker_web/api.py]: you're right, just changed to lower case

**Pull request merged**